lib/atlassian-jwt-authentication/verify.rb

```diff
@@ -81,12 +81,7 @@ def verify
 
         qsh = Digest::SHA256.hexdigest(qsh)
 
-        unless data['qsh'] == qsh
-          log(:error, "QSH mismatch for client_key #{data['iss']} and addon_key #{addon_key}")
```

Comment:
@bulinutza did this log line turned useful to you? Do you depend on it or can it be removed?